<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>904</width>
    <height>683</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>OpenSSL GUI</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>410</y>
      <width>871</width>
      <height>231</height>
     </rect>
    </property>
    <property name="toolTip">
    <string>Note: Only necessary if a PEM password was set</string>
    </property>
    <property name="title">
    <string>Convert *.cer certificate to a *.pfx certificate</string>
    </property>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>PEM-Password (key):</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txt_pass_key">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>50</y>
       <width>221</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
    <widget class="QLineEdit" name="txt_pass_2">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>150</y>
       <width>221</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>150</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Repeat PFX password:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="convert_pfx">
     <property name="geometry">
      <rect>
       <x>640</x>
       <y>90</y>
       <width>161</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Convert to .PFX</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txt_pass_1">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>100</y>
       <width>221</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>100</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>PFX export password:</string>
     </property>
    </widget>
    <widget class="QToolButton" name="toolButton">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>50</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>?</string>
     </property>
    </widget>
    <widget class="QToolButton" name="toolButton_2">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>100</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>?</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>200</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>V 0.1.0</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>851</width>
      <height>361</height>
     </rect>
    </property>
    <property name="title">
     <string>Create a new CSR (Certificate Signing Request) with OpenSSL</string>
    </property>
    <widget class="QLineEdit" name="txt_state">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>150</y>
       <width>221</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>*Country Name:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txt_org">
     <property name="geometry">
      <rect>
       <x>610</x>
       <y>100</y>
       <width>221</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>50</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>*Locality Name:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txt_country">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>100</y>
       <width>221</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>150</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Email:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txt_name">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>50</y>
       <width>221</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>State or Province Name:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>200</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Organizatoinal Unit:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txt_email">
     <property name="geometry">
      <rect>
       <x>610</x>
       <y>150</y>
       <width>221</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>100</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>*Organization:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txt_locality">
     <property name="geometry">
      <rect>
       <x>610</x>
       <y>50</y>
       <width>221</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="txt_orgunit">
     <property name="geometry">
      <rect>
       <x>610</x>
       <y>200</y>
       <width>221</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>*Name (CN):</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_new">
     <property name="geometry">
      <rect>
       <x>620</x>
       <y>270</y>
       <width>171</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Create CSR</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>250</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>PEM-password:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>300</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Repeat PEM-password:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>200</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Challenge password:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txt_challenge_pass">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>200</y>
       <width>221</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="txt_pem_1">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>250</y>
       <width>221</width>
       <height>31</height>
      </rect>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
    <widget class="QLineEdit" name="txt_pem_2">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>300</y>
       <width>221</width>
       <height>31</height>
      </rect>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
    <widget class="QToolButton" name="toolButton_3">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>250</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>?</string>
     </property>
    </widget>
    <widget class="QToolButton" name="toolButton_4">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>50</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>?</string>
     </property>
    </widget>
    <zorder>txt_state</zorder>
    <zorder>label_2</zorder>
    <zorder>txt_org</zorder>
    <zorder>label_4</zorder>
    <zorder>txt_country</zorder>
    <zorder>label_7</zorder>
    <zorder>txt_name</zorder>
    <zorder>label_3</zorder>
    <zorder>label_6</zorder>
    <zorder>label_5</zorder>
    <zorder>txt_locality</zorder>
    <zorder>txt_orgunit</zorder>
    <zorder>label</zorder>
    <zorder>btn_new</zorder>
    <zorder>txt_email</zorder>
    <zorder>label_11</zorder>
    <zorder>label_12</zorder>
    <zorder>label_13</zorder>
    <zorder>txt_challenge_pass</zorder>
    <zorder>txt_pem_1</zorder>
    <zorder>txt_pem_2</zorder>
    <zorder>toolButton_3</zorder>
    <zorder>toolButton_4</zorder>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>904</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>txt_name</tabstop>
  <tabstop>txt_country</tabstop>
  <tabstop>txt_state</tabstop>
  <tabstop>txt_locality</tabstop>
  <tabstop>txt_org</tabstop>
  <tabstop>txt_email</tabstop>
  <tabstop>txt_orgunit</tabstop>
  <tabstop>txt_challenge_pass</tabstop>
  <tabstop>txt_pem_1</tabstop>
  <tabstop>txt_pem_2</tabstop>
  <tabstop>btn_new</tabstop>
  <tabstop>txt_pass_key</tabstop>
  <tabstop>txt_pass_1</tabstop>
  <tabstop>txt_pass_2</tabstop>
  <tabstop>convert_pfx</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
